pkg/juicefs/mount/builder/pod.go

```diff
@@ -194,3 +204,20 @@ func (r *Builder) getInitContainer() corev1.Container {
 	})
 	return container
 }
+
+func (r *Builder) getMetricsPort() int32 {
```

Comment:
Can you add unit test for this function?

Reply:
Sorry for the delay on this 

unit test added